lib/Ohmbrewer_Thermostat.cpp

```diff
@@ -323,7 +331,7 @@ int Ohmbrewer::Thermostat::doDisplay(Ohmbrewer::Screen *screen) {
     // Print the section title
     screen->print("== Thermostat #");
     screen->print(getID());
-    screen->println(" ==");
+    screen->print("  ==");
```

Comment:
Looks like we do the same thing with RIMS, and both appear to look right, so I'm gonna leave it as-is.